docs/current/sdk/elixir/043817-install.md

```diff
@@ -0,0 +1,19 @@
+---
+slug: /sdk/elixir/043817/install
+---
+
+# Installation
+
+:::note
+This Dagger Elixir SDK requires Elixir 1.14 or later with Erlang OTP 23 or later. Using Erlang OTP version 25 is recommended.
```

Comment:
I would link here to installation instructions, same as all other SDKs.

```mardown
This Dagger Elixir SDK requires [Elixir 1.14 or later](https://elixir-lang.org/install.html) with [Erlang OTP 23 or later](https://www.erlang.org/downloads). Using Erlang OTP version 25 is recommended.
```

Reply:
Updated!